extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcIdentityProvider.java

```diff
@@ -59,6 +60,13 @@ public void handle(AsyncResult<AccessToken> event) {
                     return;
                 }
                 AccessToken token = event.result();
+                try {
+                    OidcUtils.validateClaims(config.token, token.accessToken());
+                } catch (OIDCException e) {
+                    result.completeExceptionally(new AuthenticationFailedException());
```

Comment:
This should log at debug level or include the cause.

Reply:
@stuartwdouglas sure, I forgot to update it to include the cause

Reply:
done